website/docs/r/kusto_cluster.html.markdown

```diff
@@ -94,6 +96,16 @@ An `identity` block supports the following:
 
 ---
 
+A `callout_policy` block supports the following:
+
+* `callout_type` - (Required) The type of callout service. Possible values are `azure_digital_twins`, `azure_openai`, `cosmosdb`, `external_data`, `genevametrics`, `kusto`, `mysql`, `postgresql`, `sandbox_artifacts`, `sql`, and `webapi`.
+
+* `callout_uri_regex` - (Required) A regular expression or FQDN pattern for the callout URI.
```

Comment:
Suggest to reword to below as FQDN doesn't contain the *scheme* component while in your example you actually used a URI:


```suggestion
* `callout_uri_regex` - (Required) A regular expression or the callout URI.
```